client/signup/steps/theme-selection/index.jsx

```diff
@@ -126,9 +122,9 @@ const ThemeSelectionStep = React.createClass( {
 				</div>
 				<div className={ themesWrapperClassName } >
 					<StepWrapper
-						fallbackHeaderText={ this.translate( 'Choose a theme.' ) }
-						fallbackSubHeaderText={ this.translate( 'No need to overthink it. You can always switch to a different theme later.' ) }
-						subHeaderText={ this.translate( 'Choose a theme. You can always switch to a different theme later.' ) }
+						fallbackHeaderText={ this.props.translate( 'Choose a theme.' ) }
```

Comment:
Maybe it would be worth extracting `translate` above to avoid repeating `this.props` whenever we want to call it.

```javascript
const { translate } = this.props
```